inst/css/filter-panel.css

```
@@ -64,6 +64,10 @@
   height: 3rem;
 }
 
+.text-primary.mb-4 {
```

```suggestion
#teal-main_ui-filter_panel .text-primary.mb-4
```

How about explicitly targeting the filter panel?

---

Let's avoid introducing an extra class just for this. I concur with @chlebowa; I haven't identified any unforeseen repercussions elsewhere within this package.

---

So, if you're okay with the change, can you approve it?

---

while looking at other places where same class is used I realised all the titles are under label tag. and  for title 'Active Filter Variables' its under span. please refer below comment for suggested changes

---

There seems to be a reason for doing it differently for this because It is wrapped inside an additional div with the class `filter-panel-active-header` which has a flex property. So it's better not to mess with this.